### DISCUSSION TITLE
Css design

### DISCUSSION THREAD

**Discussion by tina86-hub** [searchspring/snap#883] [Q&A] [answered]
Hi,

This is my first time integrating search spring and I am following Self-snap guide

https://searchspring.github.io/snap/#/start-setup.

So far it's working, and now I am trying to changes the css of search results page and suggestions. 
Where/How can I edit the default css for these sections.

Thanks

**Comment by korgon** [comment, marked as answer]
Hi @tina86-hub,

The default CSS for components from the [library](https://searchspring.github.io/snap/packages/snap-preact-components/docs/) are not able to be modified. These default styles are intended to provide a basic "skeletal" styling to get things laid out properly and usable. Using additional stylesheets you can target the components elements to add styling on top of our defaults as needed.

If you want to disable these default styles and write your own styles completely from scratch, each component in the library supports a `disableStyles` prop that can be utilized.

Please let us know if you have any other questions regarding styling, we're happy to assist.

Cheers.

**Reply by tina86-hub** [reply]
Thanks @korgon. Is it possible to drive collections pages through SearchSpring using Snap? We are using Shopify and would like to load the collections pages through SearchSpring so that we can have similar UI, filters ,grids etc. for all the pages.

**Reply by korgon** [reply]
Absolutely! You can do this by ensuring that the background is properly set on the collection pages.
https://searchspring.github.io/snap/#/integration-backgroundFilters

You will of course need to ensure that the search controller targets appear on these category pages - typically by modifying the collection page templates within Shopify. Unfortunately, we do not currently have platform specific integration instructions in our documentation - but if you are familiar with Shopify's liquid setting the script context variables should be fairly straightforward.

**Reply by tina86-hub** [reply]
Thanks @korgon. I am facing another issue on search results page. I am trying to display quick add and quick shop buttons on product grid , but event handlers ( from theme.js file) doesn't seem to be working when search results are loaded. Is there a way to (through configuration/context) to enable these, or any other work around this.

![Screenshot 2023-09-18 at 2 18 41 PM](https://github.com/searchspring/snap/assets/59583551/25c6d04e-3f91-4090-846c-c7f5ad3cb794)

**Reply by korgon** [reply]
Typically the scripts that attach this functionality will run on page load (or shortly thereafter) - due to the asynchronous nature of our integration, you will need to manually re-trigger those script hooks. Unfortunately I am unable to provide details on how you might do that, as this can vary by site, library, script, etc... You may need to expose those functions globally (on the window) to do so.

Hope that helps.
